dapps/digital-twin-detail/src/components/overview/DetailOverviewTransactions.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="content-card">
+    <h2 class="card-heading">
+      {{ '_twin-detail.overview.transactions-title' | translate }}
+    </h2>
+
+    <evan-loading
+      v-if="!transactions"
+    />
+    <table
+      v-else
+      class="mt-4 simple hasHover"
+    >
+      <thead>
+        <th>
+          {{ '_twin-detail.overview.amount' | translate }}
+        </th>
+        <th>
+          {{ '_twin-detail.overview.name' | translate }}
+        </th>
+        <th>
+          {{ '_twin-detail.overview.date' | translate }}
+        </th>
+        <th class="action" />
+      </thead>
+      <tbody>
+        <tr
+          v-for="row in transactions"
+          :key="row.blockHash"
+        >
+          <td class="amount">
+            {{ row.feeInEve }}
+          </td>
+          <td>{{ row.initiator }}</td>
+          <td>{{ row.timestamp || 'TODO' }}</td>
+          <td>
+            <evan-button
+              size="sm"
+              type="icon-secondary"
+              icon="mdi mdi-chevron-right"
+              @click="routeToTransactionExplorer(row.blockHash)"
```

Comment:
Would be nice, if we just use the `href` attribute here, so the use can also copy the link directly by right clicking. `evan-button` supports this out of the box :)

Reply:
Cool, added it.